src/Mod/Assembly/CommandCreateSimulation.py

```diff
@@ -0,0 +1,1019 @@
+            )
+        )
+
+        self.help_label0.setWordWrap(True)
+        self.help_label1.setWordWrap(True)
+        self.help_label2.setWordWrap(True)
+        self.help_label3.setWordWrap(True)
+        self.help_label4.setWordWrap(True)
+        self.help_label5.setWordWrap(True)
+        self.help_label6.setWordWrap(True)
+        self.help_label7.setWordWrap(True)
+
+        width = 1000
+        self.help_label0.setFixedWidth(width)
+        self.help_label1.setFixedWidth(width)
+        self.help_label2.setFixedWidth(width)
+        self.help_label3.setFixedWidth(width)
+        self.help_label4.setFixedWidth(width)
+        self.help_label5.setFixedWidth(width)
+        self.help_label6.setFixedWidth(width)
+        self.help_label7.setFixedWidth(width)
+
+        self.help_label0.setVisible(False)
+        self.help_label1.setVisible(False)
+        self.help_label2.setVisible(False)
+        self.help_label3.setVisible(False)
+        self.help_label4.setVisible(False)
+        self.help_label5.setVisible(False)
+        self.help_label6.setVisible(False)
+        self.help_label7.setVisible(False)
+
+        self.help_label1.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        self.help_label2.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        self.help_label3.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        self.help_label4.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        self.help_label5.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        self.help_label6.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        self.help_label7.setTextInteractionFlags(Qt.TextSelectableByMouse)
+        # Create the Help button
+        self.help_button = QPushButton(translate("Assembly", "Help"), self.dialog)
+
+        # Slot to toggle help visibility and button text
+        def toggle_help():
+            if self.help_label1.isVisible():
+                self.help_label0.setVisible(False)
+                self.help_label1.setVisible(False)
+                self.help_label2.setVisible(False)
+                self.help_label3.setVisible(False)
+                self.help_label4.setVisible(False)
+                self.help_label5.setVisible(False)
+                self.help_label6.setVisible(False)
+                self.help_label7.setVisible(False)
+                self.help_button.setText(translate("Assembly", "Help"))
+            else:
+                self.help_label0.setVisible(True)
+                self.help_label1.setVisible(True)
+                self.help_label2.setVisible(True)
+                self.help_label3.setVisible(True)
+                self.help_label4.setVisible(True)
+                self.help_label5.setVisible(True)
+                self.help_label6.setVisible(True)
+                self.help_label7.setVisible(True)
+                self.help_button.setText(translate("Assembly", "Hide help"))
+            self.positionDialog()
```

Comment:
```suggestion
            toggled = not self.help_label0.isVisible()

            self.help_label0.setVisible(toggled)
            self.help_label1.setVisible(toggled)
            self.help_label2.setVisible(toggled)
            self.help_label3.setVisible(toggled)
            self.help_label4.setVisible(toggled)
            self.help_label5.setVisible(toggled)
            self.help_label6.setVisible(toggled)
            self.help_label7.setVisible(toggled)
            
            if toggled:
                self.help_button.setText(translate("Assembly", "Help"))
            else:
                self.help_button.setText(translate("Assembly", "Hide help"))
                
            self.positionDialog()
```

But please test it - I wrote int on github comment.